Streetcode/Streetcode.BLL/MediatR/Timeline/TimelineItem/GetById/GetTimelineItemByIdHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using AutoMapper;
+using FluentResults;
+using MediatR;
+using Streetcode.BLL.DTO.Streetcode.TextContent;
+using Streetcode.BLL.DTO.Timeline;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+namespace Streetcode.BLL.MediatR.Timeline.TimelineItem.GetById;
+
+public class GetTransactionLinkByIdHandler : IRequestHandler<GetTransactionLinkByIdQuery, Result<TimelineItemDTO>>
+{
+    private readonly IMapper _mapper;
+    private readonly IRepositoryWrapper _repositoryWrapper;
+
+    public GetTransactionLinkByIdHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper)
+    {
+        _repositoryWrapper = repositoryWrapper;
+        _mapper = mapper;
+    }
+
+    public async Task<Result<TimelineItemDTO>> Handle(GetTransactionLinkByIdQuery request, CancellationToken cancellationToken)
+    {
+        var timelineItem = await _repositoryWrapper.TimelineRepository.GetFirstOrDefaultAsync(f => f.Id == request.Id);
```

Comment:
Please space out the statement (lucky you StyleCop setup doesn't have max-line constraint).